hack/release-notes.tmpl

```diff
@@ -0,0 +1,20 @@
+# Changelog since {{ .PreviousRevision }}
+
+{{with .NotesWithActionRequired -}}
+## Urgent Upgrade Notes
+
+### (No, really, you MUST read this before you upgrade)
```

Comment:
Redundant

Reply:
Why redundant? this are the notes that are marced with `action requiered` this is important.

Reply:
Ahh you mean the phrase.

Reply:
Ok, but it is not a header for sure. If you want to emphasize it, use `**Note:**` or something. Let's not be the guys who are using huge red text to get attention

Reply:
Also, the title above it is already saying URGENT. It somebody is dumb enough to ignore it, let them suffer